rust/worker/src/execution/orchestration/compact.rs

```diff
@@ -102,6 +102,7 @@ pub struct CompactOrchestrator {
         Option<tokio::sync::oneshot::Sender<Result<CompactionResponse, Box<dyn ChromaError>>>>,
     // Current max offset id.
     curr_max_offset_id: Arc<AtomicU32>,
+    hnsw_index_id: Option<Uuid>,
```

Comment:
Hammad and I discussed a potentially simpler solution where the compaction manager deletes temporary files after all jobs have been completed. However, at that point the manager needs to know the specific HNSW index IDs to delete, so we would have to add this plumbing anyways.